recipes/libcamera/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "libcamera" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://git.libcamera.org/libcamera/libcamera.git
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  
+requirements:
+  build:
+    - cmake
+    - ninja
+    - meson
+    - pkg-config
+    - python
+    - pybind11
+    - {{ stdlib("c") }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - libdrm
+    - libjpeg-turbo
+    - sdl2
+    - openssl
+    - libboost-devel
+    - pyyaml
+    - ply
+    - jinja2
+    - libudev
+    - libudev1
+    - gstreamer
+    - libevent
+    - libdrm
+    - libtiff
+    - qt6-main
```

Comment:
these should go in the host section, not in the build section.

Build section should only have build tools.

Reply:
ok thanks, I will move them right away.